public/app/features/alerting/unified/components/rule-editor/GrafanaEvaluationBehavior.tsx

```diff
@@ -271,6 +277,35 @@ export function GrafanaEvaluationBehavior({
           evaluateEvery={evaluateEvery}
         />
         <ForInput evaluateEvery={evaluateEvery} />
+
+        {existing && (
+          <Field htmlFor="pause-alert-switch">
+            <InputControl
+              render={() => (
+                <Stack gap={1}>
+                  <Switch
+                    id="pause-alert"
+                    onChange={(value) => {
+                      setValue('isPaused', value.currentTarget.checked);
+                    }}
+                    value={Boolean(isPaused)}
+                  />
+                  <label htmlFor="pause-alert" className={styles.switchLabel}>
+                    Pause evaluation
+                    <Tooltip
+                      placement="top"
+                      content="Pausing the rule will stop its evaluation until unpaused"
+                      theme={'info'}
+                    >
+                      <Icon tabIndex={0} name="info-circle" size="sm" style={{ marginLeft: '10px' }} />
```

Comment:
This inline css style should be moved to the getStyles, or is this done on purpose?